app/api/attendees.py

```diff
@@ -31,11 +35,26 @@ def before_post(self, args, kwargs, data):
         """
         require_relationship(['ticket', 'event'], data)
 
-        ticket = db.session.query(Ticket).filter_by(id=int(data['ticket'])).first()
+        ticket = db.session.query(Ticket).filter_by(
+            id=int(data['ticket'])
+            ).first()
```

Comment:
Let's wrap this in try-catch block and except a NoResultFound error. Please test it in the case of a random ticket id which doesn't exist.

Reply:
I tried it with a random ticket and 500 wasn't returned. Instead, I got this:
```
{
    "errors": [
        {
            "status": 422,
            "source": {
                "pointer": "/data/relationships/ticket"
            },
            "title": "Unprocessable Entity",
            "detail": "Invalid Ticket"
        }
    ],
    "jsonapi": {
        "version": "1.0"
    }
}
```
So I guess no need to wrap?

Reply:
alright then. If it works then fine. :smile: